clients/cobol-lsp-vscode-extension/src/services/copybook/downloader/CopybookDownloaderForDsn.ts

```diff
@@ -140,4 +113,38 @@ export class CopybookDownloaderForDsn extends ZoweExplorerDownloader {
 
     return true;
   }
+
+  public async hasMember(
+    profileName: string,
+    dataset: string,
+    copybookName: string,
+  ) {
```

Comment:
Please cleanup the type issues in this routine.
```suggestion
  ): Promise<boolean> {
```